cypress/integration/listingFlows/viewListing.spec.js

```diff
@@ -42,4 +42,22 @@ describe('View listing', () => {
     cy.get('@listingsModal').findByRole('button', { name: /Close/ }).click();
     cy.get('@listingTitle').should('have.focus');
   });
+
+  it('redirects when a user contacts via connect', () => {
+    cy.intercept(
+      '/search/listings?category=&listing_search=&page=0&per_page=75&tag_boolean_mode=all',
+      { fixture: 'search/listings.json' },
+    );
+
+    cy.visit('/listings');
+    cy.findByRole('link', { name: 'Another listing' }).as('listingTitle');
+    cy.get('@listingTitle').click();
+    cy.findByTestId('listings-modal').as('listingsModal');
+    cy.findByTestId('listing-new-message').type('Hello there!');
+    cy.get('@listingsModal')
+      .findByRole('button', { name: /Send/ })
+      .as('sendButton');
+    cy.get('@sendButton').focus().click();
+    cy.url().should('not.include', '/listings');
```

Comment:
![image](https://user-images.githubusercontent.com/833231/115740724-1be1c900-a35d-11eb-98eb-46da71ffbd84.png)

This test passes, but the check on the URL could be improved.

We could do for example 

`cy.url().should('include', '/connect/@admin_mcadmin');`

and also check that the page has the login form.

`cy.findByRole('main').findByTestId('login-form');`

http://localhost:3000/connect/@hegmannluigi

Reply:
Good idea, I wasn't quite sure on the right way to do this, but I like your suggestion!